docs/music-server.md

```diff
@@ -50,6 +50,7 @@ $ sox --version
 * [SoX](http://sox.sourceforge.net/): Download from http://sourceforge.net/projects/sox/files/sox/
 * [QuickTime Player](http://www.apple.com/quicktime/download/) or [iTunes](http://www.apple.com/itunes/download/)
 * [qaac](https://sites.google.com/site/qaacpage/)
+* [xampp](https://www.apachefriends.org) **For Hosting on your Local Machine**
```

Comment:
IMO, this would look better if we made it like this:

```
* [qaac]

For hosting on your local machine:

* [xampp](https://www.apachefriends.org)
```

Reply:
Fixed